src/NUnitFramework/framework/Constraints/ExactCountConstraintResult.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Text;
+
+namespace NUnit.Framework.Constraints
+{
+    /// <summary>
+    /// Contain the result of matching a <see cref="ExactCountConstraint"/> against an actual value.
+    /// </summary>
+    public sealed class ExactCountConstraintResult : ConstraintResult
+    {
+        /// <summary>
+        /// The actual value to which the Constraint was applied.
+        /// </summary>
+        private readonly object _actualValue;
```

Comment:
Redundant since the base constructor sets the `ActualValue` property to the same value.